content/en/container-orchestration.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Container Orchestration
+status: Completed
+category: Concept
+---
+
+## What it is
+[Container](/container/) orchestration provides the ability to manage and automate the lifecycle of containerized applications in dynamic environments. This includes different aspects that are also pursued by [Cloud Native Technologies](/cloud-native-tech/) and [Cloud Native Apps](/cloud-native-apps/), such as: deployment, (auto)scaling, auto healing and monitoring.
+It functions like a data center operating system (OS), managing [containerized](/containerization/) applications at scale (just like the OS on your laptop that manages your apps).
```

Comment:
I am not sure if this metaphor is accurate. I like the orchestrator of an orchestra metaphor, but that is just my opinion. 

Maybe we can say something like: "It works like a conductor of an orchestra, who makes sure that every part of the orchestra does what it should."